File: testprojects/java/benchmark/accuracy-test-2013-04-17/domain/indirect/indirectto/ServiceOne.java
package domain.indirect.indirectto;

import java.util.Calendar;

public class ServiceOne{
	public static String sName = "ServiceOne";
	public String name;
	public Calendar day;
	public POI poi;
	
	public ServiceOne() {
		name = "ServiceOne";
		day = Calendar.getInstance();
	}
	public String getName(){
		return name;
	}
	
	public static String getsName(){
		return sName;
	}
	
		public Calendar getDay(){
		return day;
	}
	
		public POI getPOI(){
			return poi; 
		}
		
}

File: src/test/resources/csharp/benchmark/Domain/Indirect/IndirectTo/ServiceOne.cs
namespace Domain.Indirect.IndirectTo
{
	public class ServiceOne
	{
		public string sName = "ServiceOne";

		public string name;

        public System.DateTime day;

		public POI poi;

		public ServiceOne()
		{
			name = "ServiceOne";
            day = System.DateTime.Now;
		}

		public string GetName()
		{
			return name;
		}

		public static string GetsName()
		{
			return new ServiceOne().sName;
		}

        public System.DateTime GetDay()
		{
			return day;
		}

		public POI GetPOI()
		{
			return poi;
		}
	}
}
